src/Examples/Flowsheets/hda_reaction.py

```diff
@@ -163,7 +170,7 @@ def build(self):
 
         self.rate_expression = Constraint(
             expr=self.reaction_rate["R1"] ==
-            self.k_rxn*self.state_ref.pressure *
+            self.k_rxn * self.state_ref.pressure *  # TODO: Determine if pressure should be in this equation
```

Comment:
Yes, pressure should be here. It is a gas phase reaction and the rate constant is defined in terms of pressure,